tsconfig.json

```diff
@@ -12,11 +12,12 @@
         "pretty": true,
         "strictNullChecks": true,
         "strict": true,
-        "lib": ["es5", "es2015", "es2016", "es2017"],
+        "lib": ["es5", "es2015", "es2016", "es2017", "dom"],
```

Comment:
Are you sure `dom` needs to be the last lib? If not - how about sorting these and making "dom" a first element in the array?